actions/README.md

```diff
@@ -0,0 +1,33 @@
+# slsa-verifier-installer GitHub Action
```

Comment:
I'm not sure what the current best practices are here but most actions (but not all) take the form `setup-*` for installing tools in the GitHub Actions workspace. The [official ones](https://github.com/actions?q=setup&type=all&language=&sort=) use this convention. I'm wondering if we should follow that convention.

@laurentsimon @asraa @naveensrinivasan WDUT?

Reply:
I agree, we should probably be implementing it is TS IMO

Reply:
re: `setup-*` that sounds like a great idea. Would we call it just `setup`? The setup actions you link to are under the `actions` org, so they need a `setup-<name>`. Since this action is for the verify, do we ned to further qualify it? If so, what would it be?

re: TS. This would be ideal. Since this code is fairly small and hash verification (super critical) is implemented in TS, how about we see how much bash code is required to make this work? If it becomes too large, we update it to TS next iteration? I feel like we can get away with just a few lines of bash for this.

Feel free to disagree, though.

Reply:
I went with ```setup``` for the name. 

I tried using the SHA256 action from slsa-framework/slsa-github-generator, and I didn't like the outcome. (You can take a look at https://github.com/kpk47/slsa-verifier/tree/aborted-wf if you're interested.) I'm happy to rewrite this action in TS since it ended up being quite a lot of bash.